### ISSUE TITLE
ion-toggle is always to the right when using RTL mode

### ISSUE THREAD

**Issue by arLevi** [ionic-team/ionic#12630]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[X] **2.x**
[ ] **3.x**

**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
I'm using `$app-direction: rtl;`
Using a simple `<ion-toggle [(ngModel)]="item.active"></ion-toggle>` - the button is always stuck at the right side

![screen shot 2017-08-09 at 11 40 03 pm](https://user-images.githubusercontent.com/1129836/29145336-630ec962-7d65-11e7-9d8c-0e261f808f4f.png)

**temporary fix I use**
The problem with this fix - is that the animation still appears on the wrong side - but only when closing ... 
```
[dir="rtl"] {
    .toggle-ios .toggle-inner {
        right: auto;
        left: 2px;
    }
}
```

**Comment by AmitMY** [comment]
In addition to `$app-direction: rtl;`, you have to set the platform direction as well.
What is the `dir` value of your `<html`> in the `index.html` file?

It is weird that you are adding `[dir="rtl"]` for  `$app-direction: rtl;`, as with that direction there are no directional properties in the css.

Please let me know, and let's fix it :)

**Comment by AmitMY** [comment]
Actually nevermind. I just now saw that you are using version 2.
We did RTL support in 3.4.0, and it is a simple upgrade process (go over the changelog since v3, and do the steps)

**Comment by jgw96** [comment]
Hello all! Since this has been fixed in 3.4.0 I am going to close this issue for now. Thanks!

**Comment by sameer05k21a0440** [comment]
Hi, using Ionic v3 need to implement ion-toggle with ion-toggle-text ?  I can't able to see result with custom text value .
```
 <ion-toggle ion-toggle-text="online;offline"  [(ngModel)]="customText" formControlName="customText">
                Custom text: <b>{{ customText || false }}</b>
              </ion-toggle>
```
